skipchain/skipchain.go

```diff
@@ -855,6 +894,16 @@ func (s *Service) forwardLinkLevel0(src, dst *SkipBlock) error {
 
 	// create the message we want to sign for this round
 	roster := src.Roster
+
+	if enableViewChange {
```

Comment:
This is a really big deal, and needs a nice big comment explaining it, I think. What are the trade-offs here? What is the security risk that skipchain allows this? etc. (I think this is the right implementation, I just think a lot of reasoning is locked in your head that needs to be in comments.)

Reply:
added a comment in the `enableViewChange` declaration